series/series_linux.go

```diff
@@ -82,10 +82,6 @@ func ReleaseVersion() string {
 	return release["VERSION_ID"]
 }
 
-func updateLocalSeriesVersions() error {
```

Comment:
There is a series_nonlinux.go version of this func as well.  Removing to call updateDistroInfo() directly cause build failure on macOS:

$ go build ./...
# github.com/juju/os/series
series/supportedseries.go:614:9: undefined: updateDistroInfo
series/supportedseries.go:627:13: undefined: updateDistroInfo

Reply:
Yeah, i'll swap it over :D